dotnet/samples/DocumentationExamples/Planner.cs

```diff
@@ -93,5 +93,11 @@ await foreach (var content in result)
 
     public Planner(ITestOutputHelper output) : base(output)
     {
+        //SimulatedInputText = [
+        //    "What is 49 divided by 37?",
+        //    "What is the square root of 49?",
+        //    "What is the sum of all numbers from 1 to 10?",
+        //    "If my investment of 2130.23 dollars increased by 23%, how much would I have after I spent $5 on a latte?"
+        //];
```

Comment:
@nkelly75 Thanks for the fix. Can you remove these commented out lines and we can get this merged.

Reply:
Thanks @markwallace-microsoft . Removed those comments -- they were only used as sample inputs while illustrating the issue.